scripts/build-images

```diff
@@ -12,7 +12,15 @@ mkdir -p dist
 
 for i in $BASE/[1-9]*; do
     name="os-$(echo ${i} | cut -f2 -d-)"
+
+    if [ "$name" == "os-docker" ]; then
+        # VERSION tag for docker engine images
+        IFS='-' read -ra dirname <<< "$i"
```

Comment:
no, again, I haven't changed anything :) this is what is in the os-engine repo :)